### PR TITLE
NTR: pediatric high-grade glioma

### LINKED ISSUE #8126: Request for new term [pediatric high-grade glioma]
### Label

pediatric high-grade glioma

### Synonyms

pHGG

### Synonym type

exact

### Definition

A malignant high-grade glioma that occurs during childhood in the central nervous system.

### Parent term

MONDO:0100342 malignant glioma

### Children term(s)

MONDO:0858939 diffuse pediatric-type high-grade glioma, H3-wildtype and IDH-wildtype

### ORCID Identifier

_No response_

### Website URL

_No response_

### Additional comments

Related terms:
- NCIT term: [Childhood Malignant Glioma](https://www.ebi.ac.uk/ols4/ontologies/ncit/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FNCIT_C202298)

Unclear if these should be additional parent terms:
- MONDO:0006517 - [childhood malignant neoplasm](http://purl.obolibrary.org/obo/MONDO_0006517)
- MONDO:0021079 - [childhood neoplasm](http://purl.obolibrary.org/obo/MONDO_0021079)

Preprint/Publication:
- https://www.biorxiv.org/content/10.1101/2024.03.06.583588v1
- [10.3390/genes13040624](https://doi.org/10.3390%2Fgenes13040624)

### PR DESCRIPTION
**Pull request by katiermullen**
Add new term 'pediatric high-grade glioma'

Closes #8126

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -552390,7 +552390,8 @@ subset: rare
 xref: DOID:0081277 {source="MONDO:equivalentTo"}
 xref: MEDGEN:1808288 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: UMLS:C5669918 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1808288"}
-is_a: MONDO:0100342 {source="DOID:0081277"} ! malignant glioma
+is_a: MONDO:1010030 {source="DOID:0081277"} ! pediatric high-grade glioma
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:552393]: DO does not say that this term is a child of pediatric high-grade glioma. 
Please keep the original SubClass of as it is the actual SC according to DO (it is redundant, but we like redundancy in Mondo :-) it also helps identify potential errors). 

Fro this new SC (pediatric high-grad glioma), please add the orcid

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -561130,6 +561131,17 @@ intersection_of: MONDO:0100332 NCBITaxon:28314 ! disease has primary infectious
...
+id: MONDO:1010030
+name: pediatric high-grade glioma
+def: "A malignant high-grade glioma that occurs during childhood in the central nervous system." [PMID:35456430, PMID:38496580]
+synonym: "childhood malignant glioma" EXACT [NCIT:C202298]
+synonym: "pHGG" EXACT ABBREVIATION [PMID:35456430, PMID:38496580]
+xref: NCIT:C202298 {source="MONDO:equivalentTo"}
+is_a: MONDO:0006517 {source="https://orcid.org/0000-0002-5002-8648"} ! childhood malignant neoplasm
+is_a: MONDO:0100342 {source="https://orcid.org/0000-0002-5002-8648"} ! malignant glioma
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-5002-8648
+
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:561144]: please add link to the issue.

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@katiermullen 
Please see comments in line. Thank you.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:552393]: Thanks for catching! Added back the original SC according to DO -> malignant glioma. 

For the new SC pediatric high-grade glioma, I added the orcid as the source.

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:561144]: added!

**Comment by katiermullen** [comment]
@sabrinatoro thank you for the review! I have updated the PR based on your insightful comments.

**Review by sabrinatoro — APPROVED** [review]

**Pull request merged**